recipes-multimedia/gstreamer/gstreamer1.0-plugins-nvvideo4linux2_1.14.0-r32.5.0.bb

```diff
@@ -27,6 +27,7 @@ DEPENDS = "gstreamer1.0 glib-2.0 gstreamer1.0-plugins-base virtual/egl tegra-lib
 PACKAGECONFIG ??= "libv4l2"
 PACKAGECONFIG[libv4l2] = ",,v4l-utils,tegra-libraries-libv4l-plugins"
 EXTRA_OEMAKE = "${@bb.utils.contains('PACKAGECONFIG', 'libv4l2', 'USE_LIBV4L2=1', '', d)}"
+TARGET_CFLAGS += "-fcommon"
```

Comment:
You can just use `CFLAGS` here, instead of `TARGET_CFLAGS`.  Probably doesn't matter much for this recipe, but that's the more common usage.